packages/components/src/Pagination/PageButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useRef } from 'react';
+import { useButton } from '@react-aria/button';
+import { useClassNames } from '@marigold/system';
+
+interface PageButtonProps {
+  page: number;
+  isSelected?: boolean;
```

Comment:
I know they are internal but we should follow here also our guidelines on prop naming conventions `selected` instead of `isSelected`

Reply:
same for other

Reply:
If this really a guideline I wouldn't agree to this, because using `isSelected `is more descriptive in my opinion than just `selected`. I will change it now for consistency reasons.